src/include/ceph_features.h

```diff
@@ -164,8 +164,8 @@ DEFINE_CEPH_FEATURE(59, 1, FS_BTIME)
 DEFINE_CEPH_FEATURE(59, 1, FS_CHANGE_ATTR) // overlap
 DEFINE_CEPH_FEATURE(59, 1, MSG_ADDR2) // overlap
 DEFINE_CEPH_FEATURE(60, 1, OSD_RECOVERY_DELETES) // *do not share this bit*
+DEFINE_CEPH_FEATURE(61, 1, QOS_DMC)
```

Comment:
Please do not use the last feature bit here.. instead recycle one of the newly available ones (now that luminous is out).

Reply:
for example, 16

Reply:
Thank you for comment! I used 16 as you said.